Sources/SwiftCrossUI/Views/Text.swift

```diff
@@ -14,14 +14,14 @@ public struct Text: View {
     }
 
     public func asWidget(_ children: EmptyViewContent.Children) -> GtkLabel {
-        let widget = GtkLabel()
+        let widget = GtkLabel(string: "")
```

Comment:
assuming it does smart comparison under the hood for checking draw size:
when you set it to `""` it will calculate its size for that then when you set it to `string` in the update func it has to calculate size again, and it costs us nothing to give it the string right away.
```suggestion
        let widget = GtkLabel(string: string)
```